e2e_deployment_files/dm_module_to_module_deployment.json

```diff
@@ -40,6 +40,9 @@
               "OptimizeForPerformance": {
                 "value": "<OptimizeForPerformance>"
               },
+              "MqttEventsProcessorThreadCount": {
```

Comment:
@philipktlin do we need to update the E2E scripts too?

Reply:
Yes, we need to update E2E script; otherwise it will fail. Will do.

Reply:
Talked to Anca; she showed me that current implementation in client  SDK will do nothing if this value is not int.  So leave it as is except for ARM e2e tests since we use RPi to run ARM tests.  Anca suggested to set it to 1 for ARM tests.